docs/utilities/sqs_large_message_handling.md

```diff
@@ -180,4 +180,31 @@ processing.
             return "ok";
         }
     }
+    ```
+
+## Passing custom S3Client
```

Comment:
Consider a change to `Overriding the default S3Client` to match the method name.